### ISSUE TITLE
[Chore] Add new metrics to redis exporter

### ISSUE THREAD

**Issue by oliver006** [oliver006/redis_exporter#620]
When running `INFO` I see quite a few new metrics like 
```
unexpected_error_replies:0
total_error_replies:0
dump_payload_sanitizations:0
total_reads_processed:2
total_writes_processed:1
io_threaded_reads_processed:0
io_threaded_writes_processed:0
```
and probably quite a few more.

Should add these to the exporter.

**Comment by oliver006** [comment]
Absolutely, all help is appreciated, I'll have a look at your PR soon, thanks!

**Comment by oliver006** [comment]
Closed by #621
